src/core/Akka.Remote/Serialization/ExceptionSupport.cs

```diff
@@ -137,5 +149,8 @@ public Exception ExceptionFromProtoNet(Proto.Msg.ExceptionData proto)
 
             return obj;
         }
+
+        private string ValueOrNull(string value)
+            => string.IsNullOrEmpty(value) ? null : value;
```

Comment:
Default value for strings inside an `Exception` is null, have to convert proto empty strings to null to make a proper `Exception` object